src/txs/Tx.tsx

```diff
@@ -375,7 +367,6 @@ function Tx<TxValues>(props: Props<TxValues>) {
           <dd>
             {gasFee.amount && (
               <Read
-                decimals={decimals}
                 {...gasFee}
```

Comment:
Double check decimals of `Fee` vs send token.  On branch:

<img width="404" alt="Screenshot 2023-08-18 at 10 29 40 AM" src="https://github.com/terra-money/station/assets/30275692/1bce95fb-47c4-4c6b-91e0-84cf3c19e300">


Production:

<img width="405" alt="Screenshot 2023-08-18 at 10 30 15 AM" src="https://github.com/terra-money/station/assets/30275692/d46f0519-80bb-4b7a-a4aa-187feec4f761">

Reply:
these are incorrect on prod currently and correct here